lib/danger/ci_source/ci_source.rb

```diff
@@ -2,7 +2,15 @@ module Danger
   module CISource
     # "abstract" CI class
     class CI
-      attr_accessor :repo_slug, :pull_request_id
+      attr_accessor :repo_slug, :pull_request_id, :supported_request_sources
+
+      def supported_request_sources
+        @supported_request_sources ||= []
```

Comment:
Shouldn't this raise an exception if this is meant to be implemented by all concrete CI Sources?